src/bin/svg-to-ts.ts

```diff
@@ -21,7 +24,8 @@ const DEFAULTS = {
   outputDirectory: './dist',
   prefix: 'myIcon',
   sourceFilesRegex: ['*.svg'],
-  typeName: 'myIcons'
+  typeName: 'myIcons',
+  optimizeForLayzLoading: false
```

Comment:
There seem's to be a typo here

Reply:
What typo, I don't see it ;)

Reply:
`optimizeForLayzLoading` should be `optimizeForLazyLoading` right ?

Reply:
Ah ok, yes y and z is wrong. Sorry, had a hard time seeing a difference 😅will correct it